README.md

```diff
@@ -10,6 +13,104 @@ The (evolving) protocol spec is rendered at [protocol.penumbra.zone][protocol].
 
 The (evolving) API documentation is rendered at [rustdoc.penumbra.zone][rustdoc].
 
+To participate in our test network, [keep reading below](#getting-started-on-the-test-network).
+
+For instructions on how to set up a node, [jump down and read on](#running-a-penumbra-node).
+
+## Getting started on the test network
+
+[Penumbra Labs][Penumbra] runs a test network for the latest version of the Penumbra protocol, and
+we would be delighted for you to try it out! Keep in mind that especially in the early stages of
+development, we expect many things to break. If you encounter bugs or wish for features to exist,
+we'd love for you to reach out to us on our [Discord server][Discord].
```

Comment:
It could be good for this to highlight more explicitly that we're planning to launch (and crash) lots of testnets, that users should expect data loss at this early stage, that this is really a glorified public CI, and that the testnets have no monetary value whatsoever, rather than waiting for the part at the bottom

Reply:
Added clarifying language to this effect.